composer.json

```diff
@@ -36,11 +36,20 @@
         "squizlabs/php_codesniffer": "~2.0",
         "vlucas/phpdotenv": "^2.0 | ^3.0 | ^4.0 | ^5.0",
         "symfony/process": ">=2.7 <6.0",
+        "codeception/module-apc": "*@dev",
```

Comment:
Do we need all these requirements for tests?

Reply:
They were required by the original docker image: https://github.com/Codeception/Codeception/blob/4.1/Dockerfile#L59

If they are actually used, I dont know, but if not, they shouldn't be in that image either? or does that image serve a special purpose?

Reply:
No idea.